modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -107,111 +107,434 @@ Status description:: error: data exception - unsupported value. The provided val
+
+// === 22N93
+
+// Status description:: error: data exception - missing YIELD. A required YIELD clause is missing.
+
+// === 22N94
+
+// Status description:: error: data exception - invalid YIELD *. 'YIELD *' is not supported in this context. Explicitly specify which columns to yield. 
+
+=== 22N95
+
+Status description:: error: data exception - parsing JSON exception. Invalid JSON input. Please check the format.
+
+=== 22N96
+
+Status description:: error: data exception - mapping JSON exception. Unable to map the JSON input. Please verify the structure.
+
+=== 22N97
+
+Status description:: error: data exception - unexpected struct tag. Unexpected struct tag: `{ $tagStringAsHex }`.
+
+=== 22N98
+
+Status description:: error: data exception - wrong first field during deserialization. Unable to deserialize request. Expected first field to be `{ %s }`, but was '{ %s }'.
+
+=== 22N99
+
+Status description:: error: data exception - wrong token during deserialization. Unable to deserialize request. Expected `{ %s }`, found `{ %s }`.
+
+=== 22NA0
+
+Status description:: error: data exception - invalid property based access control rule. Failed to administer property rule.
+
+// === 22NA1
+
+// Status description:: error: data exception - invalid property based access control rule involving non-comutative expressions. The property `{ $propKey }` must appear on the left hand side of the `{ $operation }` operator.  
+
+// === 22NA2
+
+// Status description:: error: data exception - invalid property based access control rule involving multiple properties. The expression: `{ $expr }` is not supported. Property rules can only contain one property.
+
+// === 22NA3
+
+// Status description:: error: data exception - invalid property based access control rule involving NaN. 'NaN' is not supported for property-based access control.
+
+=== 22NA4
+
+Status description:: error: data exception - invalid property based access control rule involving comparison with NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'.
+
+=== 22NA5
+
+Status description:: error: data exception - invalid property based access control rule involving IS NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use `IS NULL' instead.
+
+=== 22NA6
+
+Status description:: error: data exception - invalid property based access control rule involving IS NOT NULL. The property value access rule pattern `{ $pred }` always evaluates to 'NULL'. Use 'IS NOT NULL' instead.
+
+// === 22NA7
+
+// Status description:: error: data exception - invalid property based access control rule involving nontrivial predicates. The expression: `{ $expr }` is not supported. Only single, literal-based predicate expressions are allowed for property-based access control.
+
+=== 22NA8
+
+Status description:: error: data exception - parsing JSON exception. Underlying error: `{ $cause }`.
```

Comment:
```suggestion
Status description:: error: data exception - parsing JSON failure. Underlying error: `{ $cause }`.
```

I know we changed this subcondition because it was duplicated, but looks like the spreadsheet was not updated. Also the parameter $cause is $value in the codebase, which one do we want @WilcoNeo ?

Reply:
We already have a parameter called `cause`, so maybe we should use that. I will update it in the code

Reply:
https://github.com/neo-technology/neo4j/pull/27810